vendor/github.com/spf13/cobra/fish_completions.go

```diff
@@ -113,7 +113,7 @@ function __%[1]s_clear_perform_completion_once_result
     __%[1]s_debug ""
     __%[1]s_debug "========= clearing previously set __%[1]s_perform_completion_once_result variable =========="
     set --erase __%[1]s_perform_completion_once_result
-    __%[1]s_debug "Succesfully erased the variable __%[1]s_perform_completion_once_result"
+    __%[1]s_debug "Successfully erased the variable __%[1]s_perform_completion_once_result"
```

Comment:
`go mod vendor` will remove this change. 

I know its only a typo fix, but lets remove it from this MR.

Reply:
Got it, will remove it

Reply:
Removed.